lib/Pusher.php

```diff
@@ -20,9 +20,9 @@
 		+ Phil Leggetter (phil@leggetter.co.uk)
 */
 
-class PusherException extends Exception
-{
-}
+require_once('PusherException.php');
+require_once('PusherHTTPException.php');
+require_once('TriggerResult.php');
```

Comment:
What would be the preferred time to contribute this change ? After this PR gets merged, now in master (letting you update this PR) or as a PR sent against the ``event-buffer`` branch ?

And what is your choice ? classmap or PSR-0 ?